ranger/core/linemode.py

```diff
@@ -131,3 +131,40 @@ def infostring(self, fobj, metadata):
             return '?'
         return "%s %s" % (human_readable(fobj.size),
                           datetime.fromtimestamp(fobj.stat.st_mtime).strftime("%Y-%m-%d %H:%M"))
+
+
+class HumanMtimeLinemode(LinemodeBase):
+    name = "humanmtime"
+
+    def filetitle(self, fobj, metadata):
+        return fobj.relative_path
+
+    def infostring(self, fobj, metadata):
+        if fobj.stat is None:
+            return '?'
+        return print_human_mtime(fobj.stat.st_mtime)
+
+
+class SizeHumanMtimeLinemode(LinemodeBase):
+    name = "sizehumanmtime"
+
+    def filetitle(self, fobj, metadata):
+        return fobj.relative_path
+
+    def infostring(self, fobj, metadata):
+        if fobj.stat is None:
+            return '?'
+        size = human_readable(fobj.size)
+        return "%s %11s" % (size, print_human_mtime(fobj.stat.st_mtime))
+
+
+def print_human_mtime(file_mtime):
```

Comment:
One last remark: it does no printing actually so something like `human_readable_mtime` would be more appropriate. Possibly with a leading underscore as it's pretty internal here.